packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -204,14 +215,15 @@ Future<void> pub(
     result = 'version-solving-failed';
   } else if (code != 0) {
     result = 'failure';
+    printTrace('pub output:\n$messageBuffer');
```

Comment:
`processUtils.stream()` already echos pub's output to `printTrace()` (Though usually after filtering out a couple of things. See `_filterOverrideWarnings`.)

Reply:
removed the messageBuffer code